src/components/Overview/OverviewCard.jsx

```diff
@@ -28,15 +29,11 @@ function OverviewCard({ id, title, date_for_gig, convertDateFormat }) {
       <CardContent>
         <Typography variant="h4">{title}</Typography>
         <Typography variant="h5">{convertDateFormat(date_for_gig)}</Typography>
-        {/* Other card content */}
-        <Button
-          variant="contained"
-          color="error"
-          onClick={() => handleDelete(id)}
-        >
-          Delete
-        </Button>
+
+        <OverviewDelete onClick={() => handleDeleteClick(id)} />
```

Comment:
need to call "handleDelete" function in OverviewCard (instead of "handleDeleteClick" from OverviewDelete) so it will complete the dispatch

Reply:
Sorry fro the confusion, i'm about to push a branch and can just throw this change in there for convenience-